lua/svartafanan/main.lua

```diff
@@ -1,13 +1,14 @@
 local scrambler = require("svartafanan.scramble")
 local window = require("svartafanan.window")
+local timer = require("svartafanan.timer")
 
 local M = {}
 
 ---Main function to generate a scramble and display it in a floating window
 ---@param cube_size string
 function M.main(cube_size)
 	-- Open new temporary buffer
-	local win, buf = window.open()
```

Comment:
Yes, this makes sense. I think I'll bring this over to my other plugins that I use windows in that are this contained.